opt_maven_install.sh

```diff
@@ -33,4 +34,14 @@ else
 	  tar -xvf /tmp/hive-bin.tar.gz -C /tmp \
 	  && mv /tmp/apache-hive-${HIVE_VERSION}.redhat-${RH_HIVE_PATCH_VERSION}-bin/ \
 	  $HIVE_OUT
+
+	# Note(tflannag): In previous metering releases, we got the mysql-connector-java jar
+	# for free. Now, images use RHEL8 as the base image and in order to maintain upgrades
+	# to 4.6+ releases, curl that build from PNC (.tar.gz is also available) and move to
+	# the correct path in the destination container in the Dockerfile workflow.
+	export RH_MYSQL_CONNECTOR_PATCH_VERSION=00001
+	export MYSQL_VERSION=8.0.21
```

Comment:
Just making a note of this before I forget:
- We should move the `MYSQL_VERSION` environment variable outside of the conditional flow so we can use in both of the CI and ART installations to keep the mysql connector java jar versions in sync.